src/Compilers/CSharp/Portable/Errors/MessageID.cs

```diff
@@ -178,23 +178,6 @@ public static LocalizableErrorArgument Localize(this MessageID id)
             return new LocalizableErrorArgument(id);
         }
 
-        // Returns the string to be used in the /features flag switch to enable the MessageID feature.
```

Comment:
Please do not remove this from code from the compiler. It is intended to facilitate experimental code residing in branches of the compiler, even though the product code does not contain experiments.